tests/unit/sftp_test.py

```diff
@@ -48,6 +60,22 @@ def dst_path(file_paths_pair):
     return path
 
 
+@pytest.fixture
+def other_payload():
+    """Generate a binary test payload."""
+    uuid_name = uuid.uuid4()
+    return 'Original content: {name!s}'.format(name=uuid_name).encode()
```

Comment:
The range of supported Pythons is modern enough to let us just use f-strings here:
```suggestion
    return f'Original content: {uuid_name !s}'.encode()
```

Reply:
The lint does not like the f-strings:
```
tests/unit/sftp_test.py

  67:12    WPS305 Found `f` string
  return f'Original content: {uuid_name!s}'.encode()
         ^

WPS305: Found `f` string
  1     tests/unit/sftp_test.py
```

Reply:
Oh, we just need to disable that rule. But fine, this is out of the scope right now.

Reply:
Marking this as resolved.